src/components/UserInfo/UserInfo.scss

```diff
@@ -0,0 +1,3 @@
+a {
```

Comment:
The CSS rule for the 'a' tag is too generic and may affect other components. Consider using a BEM class for the UserInfo component to scope the styles, such as '.user-info__link'.